tests/css-epub-text-transform/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>EPUB CSS Text Transform</title>
+  </head>
+  <body>
+    <h1>text-transform Test</h1>
+
+    <p>1. This test passes if the English letters appear at the same width and orientation as the Japanese characters.</p>
```

Comment:
Like, effectively monospace (relative to each other)? If so, might be clearer if they were two separate paragraphs so you can make sure they line up.